roles/openshift_facts/vars/Fedora.yml

```diff
@@ -0,0 +1,7 @@
+---
+required_packages:
+  - iproute
+  - python3-dbus
+  - python-six
+  - PyYAML
+  - yum-utils
```

Comment:
I think if we are still using yum-utils on a Fedora host, then it should be considered a bug.